server/db/models/Order.js

```diff
@@ -0,0 +1,40 @@
+const Sequelize = require("sequelize");
+const db = require("../db");
+
+const Order = db.define("order", {
+  userId: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  status: {
+    type: Sequelize.ENUM("CART", "ORDER"),
```

Comment:
You may eventually want more possible statuses (to distinguish between order placed, vs packaged, vs sent to customer). But they can easily be added here